### ISSUE TITLE
How do I use this tool?

### ISSUE THREAD

**Issue by joelmzj** [e-reznik/LinkedIn-Learning-Downloader#1]
I do apologize for asking but how can I run your tool? I'm noob but I'm really interested in this tool.
Thanks in advice.

**Comment by e-reznik** [comment]
Have you already tried all the steps described in the [Example Usage](https://github.com/e-reznik/LinkedIn-Learning-Downloader#example-usage)?

- Do you have a LinkedIn Subscription?
- Do you have JDK installed?

Please tell me, in which step you are stuck. I will be happy to help.

**Comment by joelmzj** [comment]
Yes, I have made all the steps but I don't know how to "run the program". Do I need to use any command?

**Comment by e-reznik** [comment]
No, you don't need any commands or additional parameters.

Do you have an IDE like Eclipse or Netbeans? You can just run in from there (by clicking the green arrow).
Alternatively, you can run it from the command line: 

1. `javac Main.java`
2. `java Main`

If you get any errors, just let me know.

**Comment by enrey01** [comment]
Good morning, sorry to bother you, in eclipse, how is it executed by importing the project?
Or as??
All these doubts are due to my lack of knowledge, but the truth is that I am very interested in this project.
In any case, thank you very much.
All the best

**Comment by e-reznik** [comment]
Hello,

have you already tried all the steps described in the [Example Usage](https://github.com/e-reznik/LinkedIn-Learning-Downloader#example-usage)?

First, you need to download this program. You can [clone it from GitHub](https://www.vogella.com/tutorials/EclipseGit/article.html#exercise-clone-an-existing-repository) and create a new Git project in Eclipse or simply [download the entire project](https://stackoverflow.com/questions/6466945/fastest-way-to-download-a-github-project) as a ZIP file. If you choose to download it, you need to [import it into Eclipse](https://stackoverflow.com/questions/8611196/how-to-import-an-existing-directory-into-eclipse).
Look for the Main-method and [simply run it](https://stackoverflow.com/questions/12546606/how-to-run-a-simple-java-program-in-eclipse). Either from the menu or by the shortcut `Ctrl+F11`.

Please tell me, in which step you are stuck. I will be happy to help.

**Comment by enrey01** [comment]
Good morning, thanks for your quick answers, I find myself with a problem, eclipse does not recognize the java application, and if I do it by commands it gives an error, do not feel obliged to answer, if you do not feel like it, my knowledge is limited and not I want to disturb.
In any case, thank you very much for your attention.
Greetings
![java main](https://user-images.githubusercontent.com/48988504/111977689-514b8a80-8b03-11eb-99f3-e145e84db177.jpg)

**Comment by pangrr** [comment]
same error running javac Main.java

Main.java:10: error: cannot find symbol
        Authenticator authenticator = new Authenticator();
                                          ^
  symbol:   class Authenticator
  location: class Main
